aos/closed_loop/cutOutDonutsWithWEP.ipynb

```diff
@@ -8,10 +8,10 @@
     "# CutOutDonuts with WEP\n",
```

Comment:
<p>I'd use <code>origin='lower'</code>  for ease of reading.  And add labels. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_analysis_notebooks/pull/39/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>